include/superior_mysqlpp/extras/prepared_statement_utils.hpp

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include <superior_mysqlpp/connection.hpp>
+#include <superior_mysqlpp/exceptions.hpp>
+#include <superior_mysqlpp/prepared_statement.hpp>
+#include <superior_mysqlpp/utils.hpp>
+#include <type_traits>
+
+namespace SuperiorMySqlpp
+{
+    namespace
+    {
+        /**
+         * @brief Makes prepared statement from function's signature
```

Comment:
Please mention, that preparedStatement's ResultBindings are actually "guessed" from functions arguments.